gradle.properties

```diff
@@ -1,5 +1,5 @@
 group=org.partiql
-version=1.0.0-rc.4-SNAPSHOT
+version=1.0.0-rc.4
```

Comment:
Snapshot removed but maybe we should just remove it while we're doing the package audits? We're going to release 1.0.0 soon anyways

Reply:
Yeah that's fine by me 👍